README.md

```diff
@@ -123,20 +123,31 @@ Usage:
-  -o, --output string              output type for ftw tests (default "normal")
-      --read-timeout duration      timeout for receiving responses during test execution (default 1s)
-      --show-failures-only         shows only the results of failed tests
-  -t, --time                       show time spent per test
+      --connect-timeout duration               timeout for connecting to endpoints during test execution (default 3s)
+  -d, --dir string                             recursively find yaml tests in this directory (default ".")
+  -e, --exclude string                         exclude tests matching this Go regexp (e.g. to exclude all tests beginning with "91", use "91.*").
+                                               If you want more permanent exclusion, check the 'testoverride' option in the config file.
+  -f, --file string                            output file path for ftw tests. Prints to standard output by default.
+  -h, --help                                   help for run
+  -i, --include string                         include only tests matching this Go regexp (e.g. to include only tests beginning with "91", use "91.*").
+      --max-marker-log-lines int               maximum number of lines to search for a marker before aborting (default 500)
+      --max-marker-retries int                 maximum number of times the search for log markers will be repeated.
+                                               Each time an additional request is sent to the web server, eventually forcing the log to be flushed (default 20)
+  -o, --output string                          output type for ftw tests. "normal" is the default. (default "normal")
+      --read-timeout duration                  timeout for receiving responses during test execution (default 1s)
+      --show-failures-only                     shows only the results of failed tests
+  -t, --time                                   show time spent per test
+      --wait-delay duration                    Time to wait between retries for all wait operations. (default 1s)
+      --wait-for-connection-timeout duration   Http connection timeout, The timeout includes connection time, any redirects, and reading the response body. (default 3s)
+      --wait-for-expect-body-json string       Expect response body JSON pattern.
+      --wait-for-expect-body-regex string      Expect response body pattern.
+      --wait-for-expect-body-xpath string      Expect response body XPath pattern.
+      --wait-for-expect-header string          Expect response header pattern.
+      --wait-for-expect-status-code int        Expect response code e.g. 200, 204, ... .
+      --wait-for-host string                   Wait for host to be available before running tests.
+      --wait-for-insecure-skip-tls-verify      Skips tls certificate checks for the HTTPS request.
+      --wait-for-no-redirect                   Do not follow HTTP 3xx redirects.
```

Comment:
I looked at this in the code quickly. The check function returns an error upon redirect. We might need to handle that properly. Could you add a test for that?

I also think the explanation needs to be more clear here, as it's not obvious what will happen. You can't really "wait for no redirect". I suspect that, in case of a redirect, that is the response that will be returned.

Reply:
We are not using the `Check` func from the `Checker` interface. 🤷

Reply:
I disagree :) But I looked at the code again and the error is actually a special one from the Go HTTP library. It simply causes any redirect to be treated as the final response.

Reply:
Can you point to the code 🙏?

Reply:
This one? https://pkg.go.dev/net/http#Client.CheckRedirect

Reply:
Yes, sorry.